source/_posts/2024-12-19-voice-preview-edition-the-era-of-open-voice.markdown

```diff
@@ -0,0 +1,125 @@
+
+Since we began developing our open-source voice assistant for Home Assistant, one key element has been missing - great hardware that's simple to set up and use. Hardware that hears you, gives you clear feedback, and seamlessly fits into the home. Affordable and high-quality voice hardware will let more people join in on its development and allow anyone to *preview* the future of voice assistants today. Setting a standard for the next several years to base our development around.
+
+<p class='img'><img src='/images/blog/2024-12-vpe/vpe-packaging.png' style='border: 0;box-shadow: none;' alt="Voice Preview Edition with packaging"></p>
+
+We're launching [Home Assistant Voice Preview Edition](/voice-pe/) to help accelerate our goal of not only matching the capabilities of existing voice assistants but surpassing them. This is inevitable: They'll focus their efforts on monetizing voice, while our community will be focused on improving open and private voice. We'll support the languages big tech ignores and provide a real choice in how you run voice in your home.
+
+**The era of open, private voice assistants begins now, and we'd love for you to be part of it.**
+
+### Table of contents
+
+- [Introducing Home Assistant Voice Preview Edition](#introducing-home-assistant-voice-preview-edition)
+  - [Why Preview Edition](#why-preview-edition)
+  - [Built for Home Assistant](#built-for-home-assistant)
+  - [Advanced audio processing](#advanced-audio-processing)
+  - [Bringing choice to voice](#bringing-choice-to-voice)
+  - [Fully open and customizable](#fully-open-and-customizable)
+  - [Community-driven](#community-driven)
+- [Conclusion](#conclusion)
+  - [See what voice can do today](#see-what-voice-can-do-today)
+
+<!--more-->
+
+## Introducing Home Assistant Voice Preview Edition
+
+<img src='/images/blog/2024-12-vpe/voice-preview-edition.png' style='border: 0;box-shadow: none;' alt="Voice Preview Edition with packaging">
+
+Our main goal with Voice Preview Edition was to make the best hardware to get started with [Assist](/voice_control/), Home Assistant's built-in voice assistant. If you're already using other third-party hardware to run Assist, this will be a big upgrade. We prioritized its ability to hear commands, giving it an industry-leading dedicated audio processor and dual microphones - I'm always blown away by how well it picks up my voice around the room.
+
+Next, we ensured it would blend into the home, giving it a sleek but unobtrusive design. That's not to say it doesn't have flair. When you get your hands on Voice Preview Edition the first thing you'll notice is its premium feeling injection-molded shell, which is semi-transparent, just like your favorite '90s tech. The LED ring is also really eye-catching, and you can customize it to your heart's content from full gamer RGB to subtle glow.
```

Comment:
```suggestion
Our main goal with Voice Preview Edition was to make the best hardware to get started with [Assist](/voice_control/), Home Assistant's built-in voice assistant. If you're already using other third-party hardware to run Assist, this will be a big upgrade. We prioritized its ability to hear commands, giving it an industry-leading dedicated audio processor and dual microphones - I'm always blown away by how well it picks up my voice around the room.

Next, we ensured it would blend into the home, giving it a sleek but unobtrusive design. That's not to say it doesn't have flair. When you get your hands on Voice Preview Edition the first thing you'll notice is its premium-feeling injection-molded shell, which is semi-transparent, just like your favorite '90s tech. The LED ring is also really eye-catching, and you can customize it to your heart's content from full gamer RGB to subtle glow.
```